cardano-node/src/Cardano/Node/Tracing/Tracers/ChainDB.hs

```diff
@@ -493,8 +504,18 @@ instance ( LogFormatting (Header blk)
                | dtal == DDetailed ]
             ++ [ "events" .= toJSON (map (forMachine dtal) events)
                | not (null events) ]
+            ++ [ "tipBlockHash" .= tipBlockHash]
+            ++ [ "tipBlockParentHash" .= tipBlockParentHash]
+            ++ [ "tipBlockIssuerVerificationKeyHash" .= tipBlockIssuerVkHashText]
```

Comment:
Can you use a 3-element list here (or add those fields to the non-conditional list above), avoiding spurious uses of `(++)`?

Reply:
Ok